functional-tests/devscripts/run_tests.sh

```diff
@@ -111,16 +111,14 @@ if [[ "$PR_CHECK_BUILD" == "true" ]]; then
     echo "RH-Che test image with tag ${version} found on docker. Reusing image."
 	else
     echo "Could not found RH-Che tests image with tag ${version}."
-    curl --silent -f -lSL https://index.docker.io/v1/repositories/eclipse/che-e2e/tags/${version} > /dev/null
-    RESULT=$?
-    if [[ $RESULT != 0 ]]; then
-      echo "Could not found Che test image with tag ${version}. Building own RH-Che image based on Che image with nightly tag."
-      docker build --build-arg TAG=nightly -t e2e_tests dockerfiles/e2e-saas
-      rhche_image=e2e_tests
-    else
+    if [ -n $(curl -X GET https://quay.io/api/v1/repository/eclipse/che-e2e/tag/7.7.1/images | jq .status) ]; then
```

Comment:
do we really need to handle 7.7.1 case ?

Reply:
Damn, it should be ${version} here. I'll fix that, thank you very much for your review!